xonsh/platform.py

```diff
@@ -0,0 +1,141 @@
+""" Module for platform-specific constants and implementations """
+
+import os
+import platform
+import sys
+
+try:
+    import distro
```

Comment:
Please list distro as an optional dependency in the docs / readme. Namely here: https://github.com/scopatz/xonsh/blob/master/docs/dependencies.rst